src/Resources/skeleton/forgottenPassword/twig_request.tpl.php

```diff
@@ -0,0 +1,11 @@
+<?= $helper->getHeadPrintCode('Recover your password'); ?>
+
+{% block body %}
+    <h1>Recover your password</h1>
+
+    {{ form_start(requestForm) }}
+        {{ form_row(requestForm.<?= $email_field ?>) }}
+
+        <button class="btn">Send e-mail</button>
```

Comment:
Let's use `btn btn-primary` class, otherwise it looks weird

Reply:
Ok